### PR TITLE
update storage tests, move debian-testing from storaged to udisks

### PR DESCRIPTION
**Pull request by martinpitt**
It got replaced with udisks 2.6.4 (the rename/reunion). The -lvm2
component is not packaged yet due to some missing dependencies, so
disable the check-storage-lvm2 for debian-testing.

 - [x] update image
 - [x] fix missing `mkfs.*`
 - [x] fix `/run/media` on Debian/Ubuntu (it's `/media/` there)
 - [x] fix race condition in `check-storage-format TestStorage.testFormatTooSmall`
 - [x] fix check_storage_* tests

### CODE REVIEW

**Comment by martinpitt** [comment]
bot: Image refresh for debian-testing

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-8470af0bdc1cdfd77206dae527e30bde078807db/

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-8.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-8470af0bdc1cdfd77206dae527e30bde078807db/

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-11.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-8470af0bdc1cdfd77206dae527e30bde078807db/

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by martinpitt** [comment]
bot: Image refresh for debian-testing

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-7.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-8470af0bdc1cdfd77206dae527e30bde078807db/

**Comment by cockpituous** [comment]
Image creation failed.

**Comment by martinpitt** [comment]
bot: Image refresh for debian-testing

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-9.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-85a102d47d3e58835747c073b78eccaec3a71128/

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-debian-testing-85a102d47d3e58835747c073b78eccaec3a71128

**Comment by martinpitt** [comment]
bot: Image refresh for debian-testing

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-9.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-fd3c75af162cad7460d974d7e20c6a3b45f1b8ab/

**Comment by cockpituous** [comment]
Image creation for debian-testing in process on cockpit-8.
Log: http://fedorapeople.org/groups/cockpit/logs/refresh-debian-testing-fd3c75af162cad7460d974d7e20c6a3b45f1b8ab/

**Comment by cockpituous** [comment]
Image creation aborted

**Comment by cockpituous** [comment]
Image creation done: https://github.com/cockpituous/cockpit/commits/refresh-debian-testing-fd3c75af162cad7460d974d7e20c6a3b45f1b8ab

File: test/images/scripts/debian-testing.setup [hunk 1]
```diff
@@ -48,14 +47,15 @@ virtinst \
...
-# For storaged
+# For latest udisks
 echo "deb http://deb.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
 # FIXME: we need to enable jessie-backports until docker.io gets back into testing
 echo "deb http://deb.debian.org/debian jessie-backports main" >/etc/apt/sources.list.d/jessie-backports.list
 apt-get -y update
 
 export DEBIAN_FRONTEND=noninteractive
 eatmydata apt-get -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+eatmydata apt-get -y install -t experimental udisks2
```
Comment (mvollmer) [inline test/images/scripts/debian-testing.setup]: Why is this necessary?  We didn't need `-t experimental` for storaged, no?

Comment (martinpitt) [inline test/images/scripts/debian-testing.setup]: Because storaged was *only* in experimental, so apt  didn't have any other candidate for it. udisks2 is also in testing (version 2.1.8), so by default it installs that one (testing/unstable have a higher "pin value" than experimental, i. e. you have to explicitly choose packages from experimental that you want).

Note that the image, or our tests are still broken, so don't bother reviewing this yet. I just did some initial bits on the side, but I'm working on #6252 today (this PR can wait until next week).

Comment (mvollmer) [inline test/images/scripts/debian-testing.setup]: Ahh, I see why it is necessary.  Would it make sense to introduce a COCKPIT_EXPERIMENTAL_DEPS variable instead of hard coding udisks2 here?  That would make things crystal clear.

**Comment by martinpitt** [comment]
I pushed fixes for all previous failures, debian-testing is happy now. Running tests on the other OSes now as this touched the storage tests.

**Comment by martinpitt** [comment]
@mvollmer, can I ask you to review this, as it's storaged related?

**Comment by martinpitt** [comment]
The check-sos failure is unrelated, and it seems to have gotten much more flaky recently. Everything else is green now.

**Comment by mvollmer** [comment]
> The check-sos failure is unrelated, and it seems to have gotten much more flaky recently.

On first look, creating the report just seems to take very long.  There have been bugs in sosreport that causes it to run into a five minute timeout.  Maybe something like this is happening again.
